src/__tests__/pinoLoggerOptionsBuilder.spec.ts

```diff
@@ -86,6 +86,28 @@ describe("PinoLoggerOptionsBuilder", () => {
     expect(JSON.parse(stdout.output)).toMatchSnapshot();
   });
 
+  it("must log error with extra undefined argument", () => {
+    const error = new Error("some error");
+    error.stack = "Error: some error";
+
+    const extraArguments = [undefined];
```

Comment:
What happen if  extraArguments =['arg', undefined] ?

Reply:
in this case, hash of 'arg' must be calculated as `message.msgTemplateHash`

nest upper than 9.3.0 insert `undefined` as first param in case, when we log error object without additional parameters in log.error() and if we use pino in nest with config from this factory, `calculateHash` function throw error when try to calculate hash for msgTemplateHash(which is be undefined)